lapce-data/src/config.rs

```diff
@@ -669,7 +669,7 @@ impl Config {
     }
 
     fn default_settings() -> config::Config {
-        config::Config::default()
+        Config::default()
```

Comment:
It is config::Config here.

Reply:
What's the difference? I just removed what IJ told me that is an "unnecessary qualifier"

Reply:
I see now that it's a false positive of JB 🙁 
When not specifying `config::Config`, the compiler goes to `lapce_data::config::Config` instead